openspec/changes/role-type-facet/design.md

```
@@ -0,0 +1,161 @@
+Production measurement, 3,148,859 live postings:
+
+| | Postings |
+|---|---|
+| unambiguous management marker in title | 171,726 |
+| …reachable via `category=management` | 15,760 |
+| …reachable via `seniority` in (`lead`,`c_level`) | 46,330 |
+| …reachable via either | 59,940 |
+| …reachable via neither | 111,786 |
+| titles containing "manager" | 378,410 |
+| …of which are IC roles (product/project/account/…) | 150,779 |
+| IC-ladder titles (`staff|principal|…engineer`) | 7,802 |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)